clients/web/static/lib/bootstrap/css

```diff
@@ -1 +0,0 @@
-../../built/
```

Comment:
@zachmullen What will it break? I've included symlinks (to the new file, or equivalent) for all the removed public-facing static files.